src/ddscxx/include/org/eclipse/cyclonedds/core/cdr/cdr_stream.hpp

```diff
@@ -159,6 +168,22 @@ class OMG_DDS_API cdr_stream {
     */
     size_t align(size_t newalignment, bool add_zeroes);
 
+    /**
+    * Returns the current status of serialization.
+    */
```

Comment:
The indentation is off here, but the reason for this comment is the lack of Doxygen validity. It might be that this generates a warning, but it's at least good practice to use the proper keywords for the sake of better documentation and unity. The brief summary (one line header), should use `@brief`. If a longer summary makes sense, there's a newline followed by the respective summary. If there are parameters, use then list those in the form `@param[in/out] name brief description`. The parameters are followed by `@returns <brief description>`. If there's a limited number of possible return codes, each is listed in a `@retval` line. All of this is a bit nitpicky, but since we're in the process of getting documentation in order, giving a heads up for future PRs seems like the right thing to do.

Reply:
I will go over the current 'doxygen-style' comments and make them in order.